BraveRewardsUI/User Wallet/ConnectUserWalletButton.swift

```diff
@@ -0,0 +1,72 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import UIKit
+import pop
+
+/// The button to place on the wallet header when the user hasn't currently
+/// connected with Uphold
+class ConnectUserWalletButton: UIControl {
+  
+  private let backgroundView = UIView().then {
+    $0.appearanceBackgroundColor = UIColor(hex: 0x4c54d2)
```

Comment:
We can possibly use color literals? Or do you prefer hex?

Reply:
I think we use hex everywhere in this module so keeping it consistent